### ISSUE TITLE
First world problems- columns

### ISSUE THREAD

**Issue by neilcarberry** [Nexus-Mods/Nexus-Mod-Manager#108]
seriously why on earth have your columns NEVER worked! WHY can you never actually drag them? and why is it that you can never reliable change the size of the columns! I wanna be able to see what the mods name is and the version in the same screen. I don't wanna open up nexus and then have to resize EVERY SINGLE COLUMN to get the mods version sitting beside the name!

**Comment by wgridenour** [comment]
i agree i have been having to re-download several mods at a time because its keeps losing id's/names/categories (filed in another section) and i want to see the name of the mod not all that other stuff when i have to go in a re-download 30 mods because of missing information, since I'm trying to download mods that are missing everything (even the name sometimes) that's the column i want to lock the size on, not the other six which do me absolutely no good because they are blank due to missing information.
